WordPress/WordPressUITests/Utils/XCTest+Extensions.swift

```diff
@@ -10,6 +10,7 @@ extension XCTestCase {
         continueAfterFailure = false
 
         let app = XCUIApplication()
+        app.terminate()
```

Comment:
Besides resetting the simulator before execution of all tests, I also terminate the application before every test (`app.activate()` two lines below will launch the application in case if it's not running).